packages/medusa/src/api/admin/fulfillment-sets/validators.ts

```diff
@@ -44,20 +44,9 @@ export const AdminUpdateFulfillmentSetServiceZonesSchema = z
   })
   .strict()
 
-export const AdminFulfillmentSetParams = createFindParams({
-  limit: 20,
-  offset: 0,
-}).merge(
-  z.object({
-    id: z.union([z.string(), z.array(z.string())]).optional(),
-    name: z.union([z.string(), z.array(z.string())]).optional(),
-    type: z.union([z.string(), z.array(z.string())]).optional(),
-    service_zone_id: z.union([z.string(), z.array(z.string())]).optional(),
-    created_at: createOperatorMap().optional(),
-    updated_at: createOperatorMap().optional(),
-    deleted_at: createOperatorMap().optional(),
-  })
-)
+createSelectParams()
```

Comment:
This seems like a mistake?

Reply:
There isn’t any list endpoint for fulfillment sets, and this was being used for endpoint that were returning a single set

Reply:
I think @sradevski just means the free-floating `createSelectParams()`. Think that can be deleted no?

Reply:
Ahh okay, sorry yea thats a mistake will remove it 👍